app/presenters/worldwide_organisation_presenter.rb

```diff
@@ -0,0 +1,12 @@
+class WorldwideOrganisationPresenter < ContentItemPresenter
+  include ContentItem::Body
```

Comment:
Are the ContentItem::Body and ContentItem::Metadata needed yet? I don't think I can see them being used. It'd be good to keep this commit / PR to just the things that are needed for the basic presentation, to make the git history a bit easier to follow (and if we have an apprentice doing the work to render the body, it'd be nice for them to work out what they need for that).

You might also want to review whether you need all of the `title_and_context` method once you've had a look at the differences in rendering between Whitehall and this.

Reply:
You were absolutely right and the Presenter is now just an empty class, as nothing was needed for now. Thanks for the pointer 👍